src/javaRosa.js

```diff
@@ -1477,6 +1477,10 @@ define([
                     }
                 }
             });
+
+            if (form) {
+                form.vellum.refreshCurrentMug();
```

Comment:
Oof... my first impression is yes. There should be a better way, but I'm not sure what problem this is solving so it's hard to suggest a better way without digging in deeper.